### DISCUSSION TITLE
YouTube 新規則

### DISCUSSION THREAD

**Discussion by Eileen050824** [1101-GS4719-C/Python-Community-GS4719-C#1950] [科技新知]
YouTube近期在台灣宣布推出1項新工具：AI內容揭露標籤，若影音內容中的逼真內容是由AI變造或合成技術創作而成，創作者都「必須」在影片資訊中清楚說明。如果未揭露這類內容，官方會主動套用創作者無法移除的標籤，降低觀眾受到危害的風險，更嚴重者會移除內容或暫停頻道。
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/160737635/0c8ec4d3-53ec-4ec9-ac44-0d99d6b5b77a)
我覺得AI的崛起，正在改變創作者表達自我的方式，不論是將創意構想寫成分鏡腳本，或是使用工具提升創作流程的效率，都可藉助AI。AI可以協助內容創作者改善工作流程，但也有不肖人士利用AI生成假影片，傳播假訊息或進行網路犯罪。不過，我相信觀眾也更加希望影片資訊能夠公開透明，並在觀影過程中，辨認出內容是否經過AI變造或模擬合成，所以我認為這個要求創作者向觀眾揭露AI相關資訊，是個兼顧雙方需求的新工具。
我相信這對Youtube圈的創作者是個不小的衝擊，也抱持這個功能的出發點是出於善意之想法，希望經過一段緩衝時間，創作者得以適應新流程和功能。除了能更有效的推動標籤適用性，同時防止不正當的AI應用行為，讓網路世代在AI輔助下更有秩序而非引發混亂！
資料來源：TechNews 科技新報

**Comment by SusuTima** [comment]
我認同YouTube 官方的作法，現在許多假影片層出不窮，雖然有些用AI影片的本質並無惡意，但是若是能正確的傳達，讓觀眾知道影片內容的虛實，某方面程度上也是在保護我們這些觀眾。

**Comment by YuanWang08** [comment]
我覺得這是因應時代變化很好的一個做法，現在AI生成影片太快了，裡面有些內容發行者根本也都不知道有錯或是沒有驗證過，給予觀眾警示這是一部AI生成的影片，讓他們先知道影片不是人做的要更有評斷的能力，我認為是一個好的做法。

**Comment by thomasxshen** [comment]
我覺得YouTube在台灣推出的AI內容揭露標籤，是一個針對AI合成或變造影音內容的透明度措施。這種政策可以幫助觀眾更好地辨識內容的來源，對於提升網絡內容的真實性和可信度具有重要意義。這樣的措施不僅保護觀眾免受可能的欺騙或誤導，也促使創作者對其使用的技術負責。

**Comment by Hahahahbla** [comment]
YouTube推出AI內容揭露標籤是一個針對AI技術應用的重要舉措，可提升內容透明度和觀眾警覺性。雖對創作者造成衝擊，但有助於防止假訊息和網路犯罪。這項舉措體現了技術與社會規範間的平衡，對網路空間的秩序和安全有正面影響。

**Comment by wanka2030** [comment]
支持揭露AI影片的製作以及加以標籤去揭示每部由AI產生的影片，錯誤以及虛假的知識有可能會讓剛接觸網路且缺乏辨識能力的小朋友有所影響，不單單是懵懵懂懂進入網路世界的老人，或是長期活網的青少年，錯誤的影片可能會推翻或誤導原有的知識點。

**Comment by yyujimmy** [comment]
YouTube推出AI內容揭露標籤這項新工具，確實對於創作者和觀眾都是一個重要的改變。這個舉措可以幫助觀眾更清楚地了解影片中是否存在由AI變造或合成的內容，從而降低受到不實訊息的風險。同時，這也提醒了創作者們在創作時應更加謹慎，避免濫用AI技術來製作不實內容。

隨著AI技術的發展，我們確實能夠看到它對於創作流程的影響。它可以幫助提高效率，改進創作品質，但同時也帶來了一些倫理和法律上的問題。因此，YouTube推出這樣的標籤工具，不僅可以保護觀眾的權益，也能夠引導創作者們更加負責任地使用AI技術。

雖然這對於一些創作者來說可能是一個挑戰，但我們也應該看到它的積極意義。透明度和公開揭露對於建立信任和保護網路生態系統至關重要，這個新工具的推出有助於實現這樣的目標。希望隨著時間的推移，這項工具能夠發揮其應有的作用，為網路世界帶來更多的秩序和透明度。

**Comment by willychiuchiu** [comment]
隨著AI技術的進步，創作者有了更多創作的可能性，但同時也引發了一些潛在的風險。這項要求創作者揭露AI相關資訊的新工具，是一個平衡創作自由和保護觀眾權益的舉措。透過公開透明的資訊，我們可以更好地應對AI在內容創作中的挑戰，建立更安全、有序的網路環境。

**Comment by KH1126** [comment]
我覺得這是很好的一個做法，使用人工智能來生成影片的確可以節省時間，但若是知識型的影片卻包含錯誤的知識，將對學習造成負面影響。因此，我同意這是一個很好的做法。

**Comment by jim1743** [comment]
我認為Youtube的這項規則很不錯，因為最近的AI生成太發達了，有時會很難辨識出影片的真實性，導致某些人會被影片的內容給誤導，因此我很支持Youtube官方新增這項措施，相信可以減少不正確知識的傳播。
